dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpReactiveStreams3.0-cdi3.0.feature

```diff
@@ -6,6 +6,6 @@ IBM-Provision-Capability: \
   osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=io.openliberty.cdi-3.0)(osgi.identity=io.openliberty.cdi-4.0)))"
 -bundles=io.openliberty.microprofile.reactive.streams.operators30.cdi
 IBM-Install-Policy: when-satisfied
-kind=beta
-edition=core
+kind=noship
```

Comment:
This change doesn't look right, mpReactiveStream-3.0 is already beta

Reply:
fixed